typings/styled-components.d.ts

```diff
@@ -95,7 +95,7 @@ export function withTheme<P extends { theme?: T; }, T>(component: React.Componen
 export function keyframes(strings: TemplateStringsArray, ...interpolations: SimpleInterpolation[]): string;
 export function injectGlobal(strings: TemplateStringsArray, ...interpolations: SimpleInterpolation[]): void;
 export function consolidateStreamedStyles(): void;
-export function isStyledComponent(target: string | Component<object>): boolean;
+export function isStyledComponent(target: string | React.Component<object>): boolean;
```

Comment:
Can this be `ReactComponent` instead of `React.Component` given that it's already imported?